README.rst

```diff
@@ -104,12 +104,21 @@ Project
 Authentication
 ^^^^^^^^^^^^^^
 
-Follow the `Google Cloud library guide <https://google-cloud-python.readthedocs.io/en/latest/core/auth.html>`_ for authentication. Alternatively, you can provide the path to a service account JSON file in ``create_engine()``:
+Follow the `Google Cloud library guide <https://google-cloud-python.readthedocs.io/en/latest/core/auth.html>`_ for authentication. 
+Alternatively, you can provide the path to a service account JSON file in ``create_engine()`` using the ``credentials_path`` parameter or 
+passing the credentials as a Python dictionary using the ``credentials_info`` parameter:
 
 .. code-block:: python
 
+    # provide the path to a service account JSON file
     engine = create_engine('bigquery://', credentials_path='/path/to/keyfile.json')
 
+    # provide credentials as a Python dictionary
+    credentials_info = {
+        "type": "service_account", 
+        "project_id": "service-account-project"
```

Comment:
In examples, my sense is that it sometimes helps to highlight for the end user (especially a novice) that they need to replace something by using a phrase that indicates they should provide the value (i.e. using the term "your").

In the following example I don't know if `service_account` is intended to be literal text or if we intend for the user to supply a value. If it is supposed to be user supplied, I think that should be clarified.

I am presuming that the `service-account-project` is intended to be user-specific, so I recommend a change such as the following:

```suggestion
        "type": "service_account", 
        "project_id": "your-service-account-project-identifier"
```